.github/workflows/test.yaml

```diff
@@ -8,6 +8,18 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-22.04
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: nautilus
+          POSTGRES_DB: nautilus
+        options: >-
```

Comment:
I don't think we need that.

Reply:
Not too sure about this part, the official documentation recommends adding it, let me test it first.

Reply:
Looks that we don't need that, I just deleted it.